src/components/checkboxSet/index.test.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { shallow, mount } from 'enzyme'
-import 'jest-styled-components'
+
```

Comment:
Could you remove this empty line from the middle of the imports, please? 🙂

There are a few in this PR, but I'll just comment on this one.

Reply:
Removed.

Moving this import to the setup "after env" files required snapshot updating of 4 files - 

```
src/components/filterContainer/filterLabel/__snapshots__/index.test.js.snap
src/components/filteredPagedCardContainer/__snapshots__/index.test.js.snap
src/components/pagedCardContainer/__snapshots__/index.test.js.snap
src/components/sponsors/components/sponsorBadge/__snapshots__/index.test.js.snap
```